components/pages/workPageLayout/WorkPageHeader.tsx

```diff
@@ -150,9 +155,24 @@ const WorkPageHeader = ({ work }: WorkPageHeaderProps) => {
             className="hyphens-auto break-words text-typo-heading-3 lg:mt-0 lg:text-typo-heading-2">
             {`${selectedManifestation?.titles?.full || ""}${!!titleSuffix ? ` (${titleSuffix})` : ""}`}
           </h1>
-          <h2 className="mt-grid-gap-2 text-typo-subtitle-sm uppercase lg:mt-7">
-            {`af ${displayCreators(work.creators, 100) || displayCreators(selectedManifestation?.contributors || [], 100)}`}
-          </h2>
+          {!!workCreators.length && (
+            <h2 className="mt-grid-gap-2 text-typo-subtitle-sm uppercase lg:mt-7">
+              {"af "}
+              {workCreators.map((creator, index) => {
+                return (
+                  <>
+                    <span
+                      key={creator}
+                      onClick={() => navigateToSearch(creator, actor, router)}
+                      className="animate-text-underline">
+                      {creator}
+                    </span>
+                    {index + 1 < workCreators.length && ", "}
+                  </>
```

Comment:
Consider making two components: `WorkAuthor` and `WorkAthours` this is something that we easily could have the need for in the future. And I think it improves the readability.

Reply:
I like that! I think two components are maybe a bit of an overkill, but I made a component called `Authors.tsx` - without the "Work" prefix because it can render creators of both a work or a manifestation.